<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-4I3MU0F" failures="1" tests="1" name="Default test(failed)" time="0.083" errors="0" timestamp="2022-08-25T10:10:01 IST">
  <testcase classname="testngpack.FirstTestngClass" name="method4" time="0.014">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testngpack.FirstTestngClass.method4(FirstTestngClass.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- method4 -->
  <testcase classname="testngpack.FirstTestngClass" name="method5" time="0.0">
    <ignored/>
  </testcase> <!-- method5 -->
  <testcase classname="testngpack.FirstTestngClass" name="method2" time="0.0">
    <ignored/>
  </testcase> <!-- method2 -->
  <testcase classname="testngpack.FirstTestngClass" name="method3" time="0.0">
    <ignored/>
  </testcase> <!-- method3 -->
  <testcase classname="testngpack.FirstTestngClass" name="method1" time="0.0">
    <ignored/>
  </testcase> <!-- method1 -->
</testsuite> <!-- Default test(failed) -->
